Bio/codonalign/codonalignment.py

```diff
@@ -13,7 +13,8 @@
 from Bio.Align import MultipleSeqAlignment
 from Bio.SeqRecord import SeqRecord
 
-from Bio.codonalign.codonalphabet import default_codon_table, default_codon_alphabet
+from Bio.codonalign.codonalphabet import default_codon_table, default_codon_alphabet, \
+                                            compare_codon_alphabet
```

Comment:
Just use two import lines rather than a slash line continuation?

Reply:
You can also use parentheses, which, according to PEP8, are prefered to using a backslash for line continuation:
```python
from Bio.codonalign.codonalphabet import (default_codon_table, default_codon_alphabet,
                                          compare_codon_alphabet)
```